cachelib/allocator/BackgroundMover-inl.h

```diff
@@ -25,12 +25,27 @@ BackgroundMover<CacheT>::BackgroundMover(
     : cache_(cache), strategy_(strategy), direction_(direction) {
   if (direction_ == MoverDir::Evict) {
     moverFunc = BackgroundMoverAPIWrapper<CacheT>::traverseAndEvictItems;
-
   } else if (direction_ == MoverDir::Promote) {
     moverFunc = BackgroundMoverAPIWrapper<CacheT>::traverseAndPromoteItems;
   }
 }
 
+template <typename CacheT>
+void BackgroundMover<CacheT>::TraversalStats::recordTraversalTime(uint64_t nsTaken) {
+  lastTraversalTimeNs_.store(nsTaken, std::memory_order_relaxed);
+  minTraversalTimeNs_.store(std::min(minTraversalTimeNs_.load(), nsTaken),
```

Comment:
Not sure if it's worth it but perhaps we could have a histogram for traversal times? There is some implementation for latency tracker already so maybe we could reuse this?

Reply:
Also, min an max updates are not atomic so we might lose some updates (i.e. the actual value of min and max is only approximate) but I don't think it matters much - maybe just add a comment about this in the description?